runtime/doc/lua.txt

```diff
@@ -678,10 +687,16 @@ supported match predicate is `eq?` (both comparing a capture against a string
 and two captures against each other).
 
 vim.treesitter.parse_query(lang, query)
-						*vim.treesitter.parse_query(()*
+						*vim.treesitter.parse_query()*
 	Parse the query as a string. (If the query is in a file, the caller
         should read the contents into a string before calling).
 
+vim.treesitter.get_query(lang, query_name)
+						*vim.treesitter.get_query()*
+	Get the runtime query named `query_name`. This internally calls
+	`parse_query`. This will "source" all the runtime and user defined
```

Comment:
We should decide on a word for `"source"` and then use it without quotes throughout the document. It can be described in `tree-sitter-query` section